### PR TITLE
read current project release

### LINKED ISSUE #818: Display the version number in the footer
screen shot 2018-03-06 at 11 22 17

In the footer, next to "Available on github", we can display the version number from this file: https://github.com/kartoza/projecta/blob/develop/django_project/.version

To be able to know between staging and production which version we are running

Sentry is already reading this file: https://github.com/kartoza/projecta/blob/develop/django_project/core/settings/prod.py#L47

### PR DESCRIPTION
**Pull request by sonlinux**
fix #818 

##  If version file contains the release tag
version_available

## In case it is empty
version_not_available

### CODE REVIEW

File: django_project/core/templatetags/version_tags.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+def version_tag(context):
+    """Reads current project release from the .version file."""
+    version_file = absolute_path('.version')
+    try:
+        with open(version_file, 'r') as file:
+            version = file.read()
+            context['version'] = version
+    except:
+        context['version'] = 'Unknown'
+    return context['version']
```
Comment (ismailsunni) [inline django_project/core/templatetags/version_tags.py]: __[Flake8]__

[W292] no newline at end of file

File: django_project/core/templatetags/version_tags.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+@register.simple_tag(takes_context=True)
+def version_tag(context):
+    """Reads current project release from the .version file."""
+    version_file = absolute_path('.version')
+    try:
+        with open(version_file, 'r') as file:
+            version = file.read()
+            context['version'] = version
+    except:
```
Comment (ismailsunni) [inline django_project/core/templatetags/version_tags.py]: __[Flake8]__

[E722] do not use bare except'

File: django_project/lesson/templatetags/lesson_tags.py [hunk 3]
```diff
@@ -14,3 +15,17 @@ def is_translation_up_to_date(value):
...
+def version_tag(context):
+    """Reads current project release from the .version file."""
+    version_file = absolute_path('.version')
+    try:
+        with open(version_file, 'r') as file:
+            version = file.read()
+            context['version'] = version
+    except:
+        context['version'] = 'Unknown'
+    return context['version']
```
Comment (ismailsunni) [inline django_project/lesson/templatetags/lesson_tags.py]: __[Flake8]__

[W292] no newline at end of file

File: django_project/lesson/templatetags/lesson_tags.py [hunk 4]
```diff
@@ -14,3 +15,17 @@ def is_translation_up_to_date(value):
...
+
+@register.simple_tag(takes_context=True)
+def version_tag(context):
+    """Reads current project release from the .version file."""
+    version_file = absolute_path('.version')
+    try:
+        with open(version_file, 'r') as file:
+            version = file.read()
+            context['version'] = version
+    except:
```
Comment (ismailsunni) [inline django_project/lesson/templatetags/lesson_tags.py]: __[Flake8]__

[E722] do not use bare except'

File: django_project/core/templatetags/version_tags.py [hunk 5]
```diff
@@ -0,0 +1,18 @@
+# coding=utf-8
+
+from django import template
+from core.settings.utils import absolute_path
+
+register = template.Library()
+
+@register.simple_tag(takes_context=True)
```
Comment (ismailsunni) [inline django_project/core/templatetags/version_tags.py]: __[Flake8]__

[E302] expected 2 blank lines, found 1

**Comment by timlinux** [comment]
Good idea thanks @sonlinux!

**Pull request merged**